packages/backend/src/graphql/schema.graphql

```diff
@@ -527,19 +519,9 @@ input CreatePaymentPointerInput {
   publicName: String
 }
 
-type Client implements Model {
+type PaymentPointerKey implements Model {
```

Comment:
I think `PaymentPointer` ([above](https://github.com/interledger/rafiki/blob/92c89a35aca2cfde7903abbef04fcfffb2b6f369/packages/backend/src/graphql/schema.graphql#L337)) should include the following to allow querying existing keys:
```
  keys(
    "Paginating forwards: the cursor before the the requested page."
    after: String
    "Paginating backwards: the cursor after the the requested page."
    before: String
    "Paginating forwards: The first **n** elements from the page."
    first: Int
    "Paginating backwards: The last **n** elements from the page."
    last: Int
  ): PaymentPointerKeyConnection
```

Reply:
- https://github.com/interledger/rafiki/issues/702